bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq

```diff
@@ -19,7 +19,7 @@ SqlOdbcPBIConnectorType = type function (
         ])
     )
     as table meta [
-        Documentation.Name = "OpenSearch"
+        Documentation.Name = "OpenSearch Project"
```

Comment:
Where are these changes here, they seem unrelated to the GitHub flows?

Reply:
According to Amazon's request we should build 2 connectors: one for `OpenSearch Project` and the second one - for `Amazon OpenSearch Service`. `OpenSearch` is not a valid name.